src/Sylius/Bundle/CoreBundle/Command/SetupCommand.php

```diff
@@ -43,9 +47,17 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $currency = $this->getContainer()->get('sylius.setup.currency')->setup($input, $output, $this->getHelper('question'));
-        $locale = $this->getContainer()->get('sylius.setup.locale')->setup($input, $output, $this->getHelper('question'));
-        $this->getContainer()->get('sylius.setup.channel')->setup($locale, $currency);
+        $currencySetup = $this->getContainer()->get('sylius.setup.currency');
```

Comment:
What stops us to use constructor over here?

Reply:
Sorry, but I didn't understand!

Reply:
I guess @diimpp wonders why we don't use via constructor injection. I don't remember whether it wasn't posibble few years ago, anyway it's not related to the PR subject. Maybe in `2.0` we refactor all commands.